### PR TITLE
Docs(web, web-twig, web-react): Document ModalHeader and ModalFooter …

### PR DESCRIPTION
**Pull request by crishpeen**
…usage recommendations #DS-1033

### Issue reference

DS-1033

---

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/lmc-eu/spirit-design-system/blob/main/CONTRIBUTING.md).
- [x] Follow the [PR Title/Commit Message Convention](https://github.com/lmc-eu/spirit-design-system/blob/main/CONTRIBUTING.md#commit-conventions).
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [x] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: packages/web-react/src/components/Modal/README.md [hunk 1]
```diff
@@ -131,6 +131,9 @@ Also, all properties of the [`<article>` element][mdn-article] and [`<form>` ele
 
 ModalHeader contains the title of the dialog and the close button.
 
+ℹ️ We strongly always providing the header, because it provides an accessible name for the dialog and a way to
```
Comment (literat) [inline packages/web-react/src/components/Modal/README.md:134]: I think a word is missing here; recommend?

File: packages/web-react/src/components/Modal/README.md [hunk 2]
```diff
@@ -189,6 +192,10 @@ secondary actions are optional.
 👉 Please note the actions are _visually_ ordered from right to left from the tablet breakpoint up. However, the
 _actual_ order in code is followed when users tab over the interface.
 
+ℹ️ We strongly recommend always providing the footer and at least a primary action. This way, users can
```
Comment (literat) [inline packages/web-react/src/components/Modal/README.md:195]: `We strongly recommend always providing` - this sounds weird to me. May I suggest another copy? `We strongly recommend that the footer should be provided in every use case and at least with the primary action button.` What do you think?

**Review by literat — COMMENTED** [review]
I have tried to suggest some modifications to the copy. They are not exactly clear to me 🤔

**Review by literat — APPROVED** [review]
LGTM 👍

File: packages/web-react/src/components/Modal/README.md [hunk 3]
```diff
@@ -189,6 +192,10 @@ secondary actions are optional.
 👉 Please note the actions are _visually_ ordered from right to left from the tablet breakpoint up. However, the
 _actual_ order in code is followed when users tab over the interface.
 
+ℹ️ We strongly recommend including the ModalFooter with at least one primary action in every use case to facilitate
+user interaction and ensure consistent spacing within the dialog. If omitting the ModalFooter is necessary,
+compensate by applying utility spacing classes to the ModalBody.
```
Comment (adamkudrna) [inline packages/web-react/src/components/Modal/README.md:197]: >  If omitting the ModalFooter is necessary, compensate by applying utility spacing classes to the ModalBody.

I miss _what_ to compensate

**Pull request merged**